src/express-server/app.ts

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import { parsedCookies, parsedQuery } from "./middlewares";
+import { productsRouter, usersRouter } from "./routes";
+
+const app = express();
+app.use(parsedCookies);
+app.use(parsedQuery);
+app.use(productsRouter);
```

Comment:
Unite routers in one file and apply it here as one file, because in future amount of routes will grow and it will rubbish your code